src/jobflow_remote/remote/data.py

```diff
@@ -96,22 +93,22 @@ def get_single_store(
     file_path = os.path.join(dir_path, total_file_name)
     if store_type == "maggma_json":
         return StdJSONStore(file_path)
-    elif store_type == "orjson":
+    if store_type == "orjson":
         return MinimalORJSONStore(file_path)
-    elif store_type == "msgspec_json":
+    if store_type == "msgspec_json":
         return MinimalMsgspecJSONStore(file_path)
-    elif store_type == "msgpack":
+    if store_type == "msgpack":
         return MinimalMsgpackStore(file_path)
-    elif isinstance(store_type, dict):
+    if isinstance(store_type, dict):
         store_type = dict(store_type)
         store_type["path"] = file_path
         store = MontyDecoder().process_decoded(store_type)
         if not isinstance(store, Store):
-            raise ValueError(
+            raise TypeError(
```

Comment:
I don't think this makes sense at all, and we are not testing if we rely on this returning a `ValueError` else where in the code.